### DISCUSSION TITLE
Welcome to Folder-locker Discussions!

### DISCUSSION THREAD

**Discussion by Albert-W** [Albert-W/Folder-locker#37] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nhnahidhasan** [comment]
"Please add an auto-lock feature after I close the folder." It will be the best Folder Locker software. Thank You.
